velox/exec/Task.h

```diff
@@ -544,6 +543,7 @@ class Task {
   std::vector<VeloxPromise<bool>> stateChangePromises_;
 
   TaskStats taskStats_;
+  std::shared_ptr<core::QueryCtx> queryCtx_;
```

Comment:
The previous position of `queryCtx_` is also ahead of `pool_`. I am curious as to why moving it close to `pool_` fixes the problem.